packages/extension/src/serviceWorker/index.ts

```diff
@@ -28,32 +28,26 @@ import { Protocol } from 'devtools-protocol';
 /**
  * Internal dependencies.
  */
-import { CookieStore } from '../localStore';
 import parseResponseCookieHeader from './parseResponseCookieHeader';
 import parseRequestCookieHeader from './parseRequestCookieHeader';
-import { getTab } from '../utils/getTab';
-import { getCurrentTab, getCurrentTabId } from '../utils/getCurrentTabId';
-import {
-  type CookieDatabase,
-  fetchDictionary,
-} from '../utils/fetchCookieDictionary';
-import isSingleTabProcessingMode from '../utils/isSingleTabProcessingMode';
-import addAuditsIssues from '../utils/addAuditsIssues';
+import { fetchDictionary } from '../utils/fetchCookieDictionary';
 import { ALLOWED_NUMBER_OF_TABS } from '../constants';
+import SynchnorousCookieStore from '../localStore/synchnorousCookieStore';
+import canProcessCookies from '../utils/canProcessCookies';
 
 let cookieDB: CookieDatabase | null = null;
-
-// Global promise queue.
-const PROMISE_QUEUE = new PQueue({ concurrency: 1 });
-
-let globalIsUsingCDP = false;
+const syncCookieStore = new SynchnorousCookieStore();
```

Comment:
Is it a safe place to instantiate the store? Is the service worker file called only once during it entire lifecycle?

Reply:
Moved this to chrome.runtime.onInstalled